<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/path/to-your-resource.png)</string>
   </property>
   <widget class="QLabel" name="title3Label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>337</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Presione Inicio para comenzar a graficar</string>
    </property>
   </widget>
   <widget class="QLabel" name="title1Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Determine el rango de frecuencia sobre el que desea hacer el estudio</string>
    </property>
   </widget>
   <widget class="QLabel" name="fmax_label">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>fmax:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fmin_label">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>fmin:</string>
    </property>
   </widget>
   <widget class="QLabel" name="title2Label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>263</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
</string>
    </property>
    <property name="text">
     <string>Seleccióne el coeficiente a plotear:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_butt_ref">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>307</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;</string>
    </property>
    <property name="text">
     <string>Reflexión</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">bG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_butt_abs">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>307</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;</string>
    </property>
    <property name="text">
     <string>Absorción</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">bG</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="f_min">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="f_max">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>190</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/Desktop/6227655df1c74.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/newPrefix/Desktop/6227655df1c74.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title1Label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Seleccióne dispositivo de entrada</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>62</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="export_cb">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>385</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Exportar data</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="t_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Bahnschrift SemiLight SemiConde&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>T:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_sample">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <zorder>img_label</zorder>
   <zorder>title3Label</zorder>
   <zorder>title1Label</zorder>
   <zorder>fmax_label</zorder>
   <zorder>fmin_label</zorder>
   <zorder>title2Label</zorder>
   <zorder>radio_butt_ref</zorder>
   <zorder>radio_butt_abs</zorder>
   <zorder>f_min</zorder>
   <zorder>f_max</zorder>
   <zorder>title1Label_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>export_cb</zorder>
   <zorder>t_label</zorder>
   <zorder>t_sample</zorder>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Medidor de coeficiente de reflexión acústico v1.2 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>435</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: qlineargradient(spread:pad, x1:0.516727, y1:0, x2:0.545, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>435</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: qlineargradient(spread:pad, x1:0.516727, y1:0, x2:0.545, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="bG"/>
 </buttongroups>
</ui>
